txpool/txpool_test.go

```diff
@@ -1209,7 +1280,8 @@ func (e *eoa) signTx(tx *types.Transaction, signer crypto.TxSigner) *types.Trans
 
 var signerEIP155 = crypto.NewEIP155Signer(100)
 
-func TestAddTxns(t *testing.T) {
+func TestAddTxs(t *testing.T) {
+	t.SkipNow()
```

Comment:
Leftover skip?

Reply:
Actually, no. The test is obsolete because of the limit. I think it's safe to drop it entirely

Reply:
Removed in [latest commit](https://github.com/0xPolygon/polygon-edge/pull/687/commits/6725f590a2faf39a463bffe424abef3489160296)